packages/fail-logs/tests/test_inject_failure_summary.py

```
@@ -0,0 +1,287 @@
+    _write_log(
+        project,
+        [
+            _record(command_kind="test", command="pytest tests/a", error_excerpt="FAILED a"),
+            _record(command_kind="test", command="pytest tests/b", error_excerpt="FAILED b"),
+            _record(command_kind="test", command="pytest tests/c", error_excerpt="FAILED c"),
+        ],
+    )
+    _run(monkeypatch, project)
+    out = capsys.readouterr().out
+    assert "[fail-logs]" in out
+    assert "×3" in out
```

Ruff (0.15.20)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)